cerberus/cerberus.py

```diff
@@ -327,6 +333,13 @@ def validate_schema(self, schema):
                             raise SchemaError(errors.ERROR_UNKNOWN_RULE % (
                                 constraint, field))
 
+    def _validate_coerce(self, coerce, field, value):
+        try:
+            value = coerce(value)
```

Comment:
ah, yeah. with a definition of methods like for checking types, these functions / then methods can be designed to consider context. i guess sooner or later someone will need that.